src/models/topic-extraction/TopicExtractionJob.ts

```diff
@@ -0,0 +1,18 @@
+import { JobStatus } from '../JobStatus';
+import { JobType } from '../JobType';
+
+/** Topic extraction job model */
+export interface TopicExtractionJob {
+    id: string;
+    status: JobStatus;
+    type: JobType;
+    created_on: string;
+    completed_on?: string;
+    metadata?: string;
+    callback_url?: string;
+    delete_after_seconds?: number;
+    failure?: string;
+    failure_detail?: string;
+    word_count?: number;
+    language?: string;
```

Comment:
Topic extraction job models do not include language
```suggestion
```

Reply:
I believe we do return language in the job details 
https://docs.rev.ai/api/topic-extraction/reference/#operation/SubmitTopicExtractionJob